src/Microsoft.Health.Dicom.Core/Configs/InstanceMetadataCacheConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Configs;
+
+public class InstanceMetadataCacheConfiguration : CacheConfiguration
+{
+    public InstanceMetadataCacheConfiguration()
+    {
+        MaxCacheSize = 1000;
```

Comment:
From my experiment with IMS, their layer 18 has 40K frames. And the file size is 3.7KB. So the max is 3.7KB*1000. (Which will not be true because there will be other layers which are lesser in no of frames). So worst case is 3.7 MB or 4MB.